src/patternfly/components/Masthead/masthead.scss

```diff
@@ -0,0 +1,234 @@
+  --pf-c-masthead--m-display-stack__main--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-masthead--m-display-stack__main--MarginRight: 0;
+  --pf-c-masthead--m-display-stack__main--before--BorderBottom: var(--pf-c-masthead__main--before--BorderBottomWidth) solid var(--pf-c-masthead__main--before--BorderBottomColor);
+  --pf-c-masthead--m-display-stack__content--Order: 1;
+  --pf-c-masthead--m-display-stack__content--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-masthead--m-display-stack__content--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-masthead--m-display-stack__content--MarginLeft: 0;
+  --pf-c-masthead--m-display-stack__content--c-nav--m-horizontal--MarginRight: calc(var(--pf-c-masthead--inset) * -1);
+  --pf-c-masthead--m-display-stack__content--c-nav--m-horizontal--MarginLeft: calc(var(--pf-c-masthead--inset) * -1);
+
+  // inline
+  --pf-c-masthead--m-display-inline--FlexWrap: nowrap;
+  --pf-c-masthead--m-display-inline__main--Order: 0;
+  --pf-c-masthead--m-display-inline__main--FlexBasis: auto;
+  --pf-c-masthead--m-display-inline__main--PaddingTop: 0;
+  --pf-c-masthead--m-display-inline__main--PaddingBottom: 0;
+  --pf-c-masthead--m-display-inline__main--MarginRight: calc(var(--pf-global--gutter--md) / 2); // split gutter
+  --pf-c-masthead--m-display-inline__main--before--BorderBottom: 0;
+  --pf-c-masthead--m-display-inline__content--Order: 0;
+  --pf-c-masthead--m-display-inline__content--PaddingTop: 0;
+  --pf-c-masthead--m-display-inline__content--PaddingBottom: 0;
+  --pf-c-masthead--m-display-inline__content--MarginLeft: calc(var(--pf-global--gutter--md) / 2); // split gutter
+  --pf-c-masthead--m-display-inline__content--c-nav--m-horizontal--MarginRight: 0;
+  --pf-c-masthead--m-display-inline__content--c-nav--m-horizontal--MarginLeft: 0;
+
+  // main border
+  --pf-c-masthead__main--before--Right: calc(var(--pf-c-masthead--inset) * -1);
+  --pf-c-masthead__main--before--Left: calc(var(--pf-c-masthead--inset) * -1);
+  --pf-c-masthead__main--before--BorderBottomWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-masthead__main--before--BorderBottomColor: var(--pf-global--palette--black-600);
+
+  // toggle
+  --pf-c-masthead__toggle--MarginRight: calc(var(--pf-global--gutter--md) / 2);
+
+  // light 100
+  --pf-c-masthead--m-light-100--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-masthead--m-light-100__main--BorderBottomColor: var(--pf-global--palette--black-200);
+
+  // light 200
+  --pf-c-masthead--m-light-200--BackgroundColor: var(--pf-global--BackgroundColor--200);
+  --pf-c-masthead--m-light-200__main--BorderBottomColor: var(--pf-global--palette--black-300);
+
+  // set layout to stack by default
+  @include pf-c-masthead--m-display-stack;
+
+  // set layout to inline at medium breakpoint
+  @media screen and (min-width: $pf-global--breakpoint--md) {
+    @include pf-c-masthead--m-display-inline;
+  }
+
+  @media screen and (min-width: $pf-global--breakpoint--xl) {
```

Comment:
if all you're doing is redefining a var, this should go up to the block where they're originally defined